sycl/plugins/cuda/CMakeLists.txt

```diff
@@ -49,6 +49,8 @@ if (MSVC)
   # by defining __SYCL_BUILD_SYCL_DLL, we can use __declspec(dllexport)
   # which are individually tagged for all pi* symbols in pi.h
   target_compile_definitions(pi_cuda PRIVATE __SYCL_BUILD_SYCL_DLL)
+  # Install stripped PDB
+  add_stripped_pdb(pi_cuda)
```

Comment:
Should this also be done for hip plugin?

Reply:
To my knowledge, the HIP plugin is not currently supported on Windows. As soon as it is, I imagine the CMake will include this entire MSVC section, adjusted for the HIP plugin.